bin/compare-bench-results.js

```diff
@@ -18,119 +17,126 @@ if (process.argv.length !== 4) {
   process.exit(1)
 }
 
-async.map(
-  process.argv.slice(2),
-  (file, cb) => {
-    fs.readFile(file, { encoding: 'utf8' }, (err, data) => {
-      if (err) {
-        return cb(err)
-      }
-
-      let parsed = null
-      try {
-        parsed = JSON.parse(data)
-      } catch (parseError) {
-        return cb(parseError)
-      }
+const errorAndExit = (err) => {
+  console.log('Failed to load files.')
+  console.log(err)
+  process.exit(2)
+}
 
-      return cb(null, parsed)
-    })
-  },
-  (err, resultFiles) => {
-    if (err) {
-      console.log('Failed to load files.')
-      console.log(err)
-      process.exit(2)
+const processFile = async (file) => {
+  try {
+    const data = await fs.readFile(file, { encoding: 'utf8' })
+    let parsed = null
+    try {
+      parsed = JSON.parse(data)
+    } catch (parseError) {
+      return errorAndExit(parseError)
```

Comment:
Since both these catches just `errorAndExit`, I don't think we need nested try's, we can just do one try/catch?